bcbio/structural/purecn.py

```diff
@@ -46,6 +52,63 @@ def run(items):
     out.append(paired.tumor_data)
     return out
 
+def _run_purecn_normaldb(paired, out):
+    """Run PureCN with normaldb and native segmentation
+       paired is one t/n pair or only t 
+    """
+    sample = utils.to_single_data(paired.tumor_data)
+    bed_file = tz.get_in(["config", "algorithm", "purecn_bed_ready"], sample)
+    sample_name = dd.get_sample_name(sample)
+    work_dir = _sv_workdir(sample)
+    rscript = utils.Rscript_cmd("r36")
+    purecn_r = utils.R_package_script("r36", "PureCN", "extdata/PureCN.R")
+    intervals = tz.get_in(["config", "algorithm", "purecn_bed_ready"], sample)
+    bam_file = dd.get_align_bam(sample)
+    # termline and somatic - just annotated and filters assigned
+    variants_vcf =  tz.get_in(["variants"], sample)[0].get("germline")
+    # in a T/N case, there is no germline file - vrn file with all variants
+    if not variants_vcf:
+        variants_vcf = tz.get_in(["variants"], sample)[0].get("vrn_file")
+    normaldb = tz.get_in(["config", "algorithm", "background", "cnv_reference", "purecn_normaldb"], sample)
+    mappingbiasfile = tz.get_in(["config", "algorithm", "background", "cnv_reference", "purecn_mapping_bias"], sample)
+    sample_coverage = tz.get_in(["depth", "bins", "purecn"], sample)
+    simple_repeat_bed = dd.get_variation_resources(sample)["simple_repeat"]
+    result_file = os.path.join(work_dir, sample_name + ".rds")
+    cmd = [ rscript, purecn_r,
+            "--out", work_dir,
+            "--tumor", sample_coverage,
+            "--sampleid", sample_name,
+            "--vcf", variants_vcf,
+            "--normaldb", normaldb,
+            "--mappingbiasfile", mappingbiasfile,
+            "--intervals", intervals,
+            "--snpblacklist", simple_repeat_bed,
+            "--genome", "hg38",
```

Comment:
This hardcodes the genome to `hg38`. It should instead use `genome_build` from the configuration.